x-pack/platform/plugins/shared/dataset_quality/public/hooks/use_integration_actions.ts

```diff
@@ -82,9 +82,9 @@ export const useIntegrationActions = () => {
     (dashboard: Dashboard) =>
       wrapLinkPropsForTelemetry(
         getRouterLinkProps({
-          href: dashboardLocator?.getRedirectUrl({ dashboardId: dashboard?.id } || ''),
+          href: dashboardLocator?.getRedirectUrl({ dashboardId: dashboard?.id ?? '' }),
```

Comment:
thanks for fixing this as it didn't make much sense previously, however this seems to might cause an issue where we will be navigating to creating a new dashboard from scratch. 
Its anyways not to be affecting this PR but will bring it over to the team.